CHANGELOG.md

```diff
@@ -3,6 +3,160 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 4.0.0-alpha.38
+
+_Jul 23, 2021_
+
+Big thanks to the 6 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Introduce row editing feature (#2098) @m4theushw
+- ⚛️ Remove unused row CSS classes (#2327) @ZeeshanTamboli
+- ⚡️ Rename `XGrid` to `DataGridPro` (#2382) @m4theushw
+- ✨ Move packages to `@mui` scope (#2341) @oliviertassinari
+- 💡 Remove `apiRef` from all `options` callback params (#2312) @DanailH
+
+This is the last alpha release. We are moving to beta in the next release, next week.
+
+### `@mui/x-data-grid@v4.0.0-alpha.38` / `@mui/x-data-grid-pro@v4.0.0-alpha.38`
+
+#### Breaking changes
+
+- [DataGrid] Remove unused row CSS classes (#2327) @ZeeshanTamboli
+  The CSS classes `.Mui-odd` and `.Mui-even` were removed from the row. Use the `:nth-child(odd|even)` selector to replace them.
+
+- [DataGridPro] Rename `XGrid` to `DataGridPro` (#2382) @m4theushw
+
+  ```diff
+  - import { XGrid } from '@mui/x-data-grid-pro';
+  + import { DataGridPro } from '@mui/x-data-grid-pro';
+  ```
+
+- [core] Move packages to `@mui` scope (#2341) @oliviertassinari
+
+  ```diff
+  - import { DataGrid } from '@material-ui/data-grid';
+  + import { DataGrid } from '@mui/x-data-grid';
+  ```
+
+  ```diff
+  - import { LicenseInfo } from '@material-ui/x-grid';
+  + import { LicenseInfo } from '@mui/x-data-grid-pro';
+  ```
+
+- [DataGrid] Remove `apiRef` from all `options` callback params (#2312) @DanailH
+
+  ```diff
+  export interface GridColumnHeaderParams {
+    * The column of the current header component.
+    */
+    colDef: GridStateColDef;
+    -  /**
+    -  * API ref that let you manipulate the grid.
+    -  */
+    - api: any;
+  }
+  ```
+
+  ```diff
+  export interface GridColumnOrderChangeParams {
+    * The old column index.
+    */
+    oldIndex: number;
+    -  /**
+    -   * API ref that let you manipulate the grid.
+    -   */
+    -  api: any;
+  }
+  ```
+
+  ```diff
+  export interface GridColumnResizeParams {
+    * The column of the current header component.
+    */
+    colDef: GridStateColDef;
+  -  /**
+  -   * API ref that let you manipulate the grid.
+  -   */
+  -  api: any;
+  ```
+
+  ```diff
+  export interface GridColumnVisibilityChangeParams {
+    * The column of the current header component.
+    */
+    colDef: GridStateColDef;
+  -  /**
+  -   * API ref that let you manipulate the grid.
+  -   */
+  -  api: any;
+  ```
+
+  ```diff
+  export interface GridRowParams {
+    * All grid columns.
+    */
+    columns: GridColumns;
+  -  /**
+  -   * GridApiRef that let you manipulate the grid.
+  -   */
+  -  api: any;
+  ```
+
+  ```diff
+  export interface GridRowScrollEndParams {
+    * The grid visible columns.
+    */
+    visibleColumns: GridColumns;
```

Comment:
Could you fix all the git diff? I have stopped suggesting changes, there are too many instances. Please replicate the output of `git diff`. 

```suggestion
     visibleColumns: GridColumns;
```

Reply:
they are now removed